.github/workflows/ci.yml

```diff
@@ -67,23 +71,14 @@ jobs:
 
       - uses: Swatinem/rust-cache@v1
 
-      - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --workspace --all-targets --all-features --release
+      - name: Build crates
+        run: .github/workflows/scripts/for_each_crate.sh "build" "--all-targets --all-features --release"
```

Comment:
Are we just checking here, whether the crates successfully build, or do we actually need the created binaries?

Reply:
Because if not, couldn't we drop the `--release` flag?

Reply:
We test later in this particular job, so by building for release here, we should be able to reuse those binaries.